### PR TITLE
Add custom labels

### LINKED ISSUE #3957: Custom Labels support for UME
### Issue Type

Missing content for an existing feature

### Links to Affected Docs

https://www.astronomer.io/docs/astro/export-metrics

### The Issue or Context

https://github.com/astronomer/astro/issues/21459
https://github.com/astronomer/astro/pull/22505
https://github.com/astronomer/astro/pull/22516

### Fix or Proposed Change

- Customers can configure custom labels to be exported with their metrics
- If they use the same label name as a default label (eg. instance), the default label will be overwritten with the custom label value
- Hosted Customers can configure custom labels if configuring UME using Environment Manager, and append Deployment specific custom labels by adding configuration at the Deployment level

### Additional Notes

To be released on August 5th

### Required Reviewers

bote795, chomipi88

### PR DESCRIPTION
**Pull request by lzdanski**
Resolves #3957

### CODE REVIEW

File: astro/universal-metrics-export.md [hunk 1]
```diff
@@ -105,17 +106,25 @@ You can enable metrics export at both the Workspace and Deployment level for Ast
...
 3. Click **+ Metrics Export** to create a new export connection.
-4. Enter the required information for your export.
-5. Click **Create metrics export**.
-6. (Optional) Allow all Deployments to link to this metrics export configuration. See [Link metrics export](#link-exports).
+4. Enter the required information for your export, the **Name** and **Endpoint**
+5. (Optional) Click **More Options** to add additional information for your endpoint:
+   - **Token** - A bearer token or license key for your Prometheus endpoint.
+   - **Username** and **Password** - Credentials for basic authentication, if needed.
+   - Custom **Headers** - Add a **Name** and **Value** pair that can be sent to your server with each remote write. If you add a custom header, any default values are overwritten by your custom configuration. You can add Deployment-specific labels when configuring **Deployment metrics**.
+6. Click **Create metrics export**.
```
Comment (bote795) [inline astro/universal-metrics-export.md:114]: do we want the same level of documentation for the labels or is it ok just like we included it at the end of the statement

**Review by bote795 — APPROVED** [review]

File: astro/universal-metrics-export.md [hunk 2]
```diff
@@ -23,6 +23,7 @@ While [Deployment Analytics](deployment-metrics.md), [Deployment health incident
 - Uses the [Prometheus data model](https://prometheus.io/docs/concepts/data_model/) format using the remote-write capability, making it flexible and easy to use.
 - Allows you to configure a metrics export at the per-Deployment level or at the Workspace level.
 - Enables you to customize your observability experience with your tooling and create custom dashboards or alerts that are not currently available in Astro.
+- Offers a way to add custom metadata as labels to your exported metrics or headers to the HTTP request.
```
Comment (merobi-hub) [inline astro/universal-metrics-export.md:26]:
```suggestion
- Offers a way to add custom metadata as labels to your exported metrics or HTTP request headers.
```

File: astro/universal-metrics-export.md [hunk 3]
```diff
@@ -105,17 +113,27 @@ You can enable metrics export at both the Workspace and Deployment level for Ast
 1. In the Astro UI, select **Environment** on the sidebar menu.
 2. Click the **Metrics Export** tab to configure your metrics export.
 3. Click **+ Metrics Export** to create a new export connection.
-4. Enter the required information for your export.
-5. Click **Create metrics export**.
-6. (Optional) Allow all Deployments to link to this metrics export configuration. See [Link metrics export](#link-exports).
+4. Enter the required information for your export, the **Name** and **Endpoint**
```
Comment (merobi-hub) [inline astro/universal-metrics-export.md:116]:
```suggestion
4. Enter the required information for your export, the **Name** and **Endpoint**.
```

File: astro/universal-metrics-export.md [hunk 4]
```diff
@@ -105,17 +113,27 @@ You can enable metrics export at both the Workspace and Deployment level for Ast
...
+5. (Optional) Click **More Options** to add additional information for your endpoint:
+   - **Token** - A bearer token or license key for your Prometheus endpoint.
+   - **Username** and **Password** - Credentials for basic authentication, if needed.
+   - Custom **Headers** - Add a **Name** and **Value** pair that can be sent to your server with as part of the HTTP request with each remote write. If you add a custom header, any default values are overwritten by your custom configuration. You can add Deployment-specific labels when configuring **Deployment metrics**.
```
Comment (merobi-hub) [inline astro/universal-metrics-export.md:120]:
```suggestion
   - Custom **Headers** - Add a **Name** and **Value** pair that can be sent to your server as part of the HTTP request with each remote write. If you add a custom header, any default values are overwritten by your custom configuration. You can add Deployment-specific labels when configuring **Deployment metrics**.
```

**Review by merobi-hub — APPROVED** [review]

**Pull request merged**